JiraPS/Public/Get-JiraUser.ps1

```diff
@@ -27,7 +27,10 @@ function Get-JiraUser {
         [Parameter()]
         [System.Management.Automation.PSCredential]
         [System.Management.Automation.Credential()]
-        $Credential = [System.Management.Automation.PSCredential]::Empty
+        $Credential = [System.Management.Automation.PSCredential]::Empty,
+
+        [Parameter()]
```

Comment:
this parameter is not relevant for `Self`.

please replace this line with
```powershell
[Parameter( ParameterSetName = 'ByInputObject' )]
[Parameter( ParameterSetName = 'ByUserName' )]
```

also: please move this above the `$Credential` parameter